server/app/api.py

```diff
@@ -439,6 +439,18 @@ def enrollment(self):
                 })
         return json.dumps(data)
 
+    def check(self, emails, course, role):
```

Comment:
Oh, this is for a theoretical "generate composition queues" based off of a set of user emails. I originally had users input a CSV, so I needed this check.